general/administration/reverse-proxy.md

```diff
@@ -197,8 +197,16 @@ server {
     }
 
     # Jellyfin
+    location /jellyfin/ {
```

Comment:
```suggestion
    location /jellyfin {
```
this is important, these two locations are different!

Reply:
fixed